utils/install-open-shift-spire.sh

```diff
@@ -7,6 +7,7 @@ TRUSTDOMAIN="spiretest.com"
 PROJECT="spire"
 SPIREGROUP="spiregroup"
 SPIREAGSA="spire-agent"
+SPIRESA="spire-server"
```

Comment:
nit: getting a bit hard to read now with the multiple different combinations, maybe rename with underscores